packages/salesforcedx-vscode-lwc/package.nls.json

```diff
@@ -18,5 +18,7 @@
   "force_lightning_lwc_test_stop_watching_all_tests_text": "SFDX: Stop Watching All Lightning Web Component Tests",
   "force_lightning_lwc_preferences": "Salesforce Lightning Web Components",
   "force_lightning_lwc_remember_device_description": "Remember most recently used mobile device target.",
-  "force_lightning_lwc_mobile_log_level": "Log level used when calling SFDX Preview on Mobile command."
+  "force_lightning_lwc_mobile_log_level": "Log level used when calling SFDX Preview on Mobile command.",
+  "force_lightning_lwc_debugger": "Debug: LWC Jest Tests",
+  "force_lightning_lwc_debugger_desc": "Debug configuretion for running LWC jest tests in vscode."
```

Comment:
typo: configuretion

Reply:
Should line 23 have "LWC jest tests" or "LWC Jest Tests"? (line 22 has "LWC Jest Tests")

Reply:
Confirmed text with @sbudhirajadoc. We're good to go.